src/test/java/br/unb/cic/analysis/samples/OverridingAssignmentLocalVariablesSample2.java

```diff
@@ -0,0 +1,16 @@
+package br.unb.cic.analysis.samples;
```

Comment:
only tests with local variables? please see my comments about the different kinds of declarations

Reply:
I did many tests with local changes with the kinds of declarations, but I will create some more.

Reply:
I added new tests for attributes and static class field for both local variables and arrays.

I did not add tests for Instance Field as this is already being resolved in Galileo's PR.